.ci/pitest-suppressions/pitest-common-suppressions.xml

```diff
@@ -378,15 +369,6 @@
     <lineContent>if (thirdPartyNameToFullModuleNames == null) {</lineContent>
   </mutation>
 
-  <mutation unstable="false">
-    <sourceFile>PackageObjectFactory.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.PackageObjectFactory</mutatedClass>
-    <mutatedMethod>createModule</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
-    <description>removed call to java/lang/Object::getClass</description>
-    <lineContent>new String[] {name, attemptedNames}, null, getClass(), null);</lineContent>
```

Comment:
This is was not being killed because of the cache.

`BUNDLE_CACHE` is built based on the `bundleName` only. The cache was pre-populated from the start up run, and then when it was requested with a `null` class it retrieved the cached `ResourceBundle`. When there is no cache entry, a NPE is generated because to generate a new `ResourceBundle` we need the classloader from the class.

With the cache removed, we need the classloader to even check `ResourceBundle`'s cache. So the NPE always occurs now.